<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCSVList</class>
 <widget class="QMainWindow" name="EditCSVList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV List Editor </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/system_software_installer.ico</normaloff>Resources/system_software_installer.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="listTable">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="verticalFrame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>124</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="addNewButton">
           <property name="text">
            <string>Add New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modifySelectedButton">
           <property name="text">
            <string>Modify Selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteSelectedButton">
           <property name="text">
            <string>Delete Selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Category</string>
           </property>
           <property name="buddy">
            <cstring>categorySel</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="categorySel">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name</string>
           </property>
           <property name="buddy">
            <cstring>nameEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="nameEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Directory</string>
           </property>
           <property name="buddy">
            <cstring>dirEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Icon</string>
           </property>
           <property name="buddy">
            <cstring>iconEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLineEdit" name="iconEdit"/>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="iconBrowse">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="1">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLineEdit" name="dirEdit"/>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="dirBrowse">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionReload"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_New"/>
    <addaction name="actionEdit_Selected"/>
    <addaction name="actionDelete_Selected"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionAdd_New">
   <property name="text">
    <string>Add New</string>
   </property>
  </action>
  <action name="actionEdit_Selected">
   <property name="text">
    <string>Modify Selected</string>
   </property>
  </action>
  <action name="actionDelete_Selected">
   <property name="text">
    <string>Delete Selected</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>categorySel</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>dirEdit</tabstop>
  <tabstop>dirBrowse</tabstop>
  <tabstop>iconEdit</tabstop>
  <tabstop>iconBrowse</tabstop>
  <tabstop>addNewButton</tabstop>
  <tabstop>modifySelectedButton</tabstop>
  <tabstop>deleteSelectedButton</tabstop>
  <tabstop>listTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>EditCSVList</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
